lib/Doctrine/Import/Builder.php

```diff
@@ -990,6 +993,52 @@ private function innerBuildActAs($actAs, $level = 0, $parent = null, array &$emi
         return $build;
     }
 
+    /**
+     * Adds the columns of the used actAs behaviors to the comment block.
+     *
+     * @param string $className
+     * @param array $instanceOptions
+     * @param array $definition
+     *
+     * @throws Doctrine_Import_Builder_Exception
+     */
+    private function addActAsColumnsToDefinition($className, $instanceOptions, &$definition)
+    {
+        if ($className && class_exists($className)) {
+            $actAsInstance = new $className($instanceOptions);
+            $options = $actAsInstance->getOptions();
+
+            if (count($options) == 0) {
+                return;
+            }
+
+            // Some behaviors do not contain an array of columns, e.g. SoftDelete.
+            if (!is_array(reset($options))) {
+                $options = [$options];
+            }
+
+            foreach ($options as $name => $column) {
+                if (!is_array($column) || !array_key_exists('name', $column) || !array_key_exists('type', $column)) {
+                    // 'name' or 'type' not found. Unfortunately there is no logger. What is the best way to abort here?
+                    continue;
+                }
+
+                if (array_key_exists('disabled', $column) && $column['disabled']) {
+                    // Column has been disabled.
+                    continue;
+                }
+
+                // Add field, if it does not exist already.
+                if (!array_key_exists($name, $definition['columns']) && !array_key_exists($column['name'], $definition['columns'])) {
+                    $definition['columns'][$name] = $column;
+                }
+            }
+        } else {
+            throw new Doctrine_Import_Builder_Exception('Missing class for actAs ' . $className . '.');
```

Comment:
This might bring a BC to existing code, we could also replace it by just emitting a warning instead of throwing an exception?

Reply:
I think that the actAs validator is not here. So just ignore that case.

Reply:
Done :)